### ISSUE TITLE
Upload image for dish should work

### ISSUE THREAD

**Issue by vishaldhiman** [vishaldhiman/vc_android_native#14]

**Comment by xamguo** [comment]
Is there an api for this?

**Comment by xamguo** [comment]
Is this finished?

**Comment by xamguo** [comment]
need the api for this

**Comment by vishaldhiman** [comment]
Added the API call for this. See:

https://github.com/vishaldhiman/vc_android_native/wiki/vChoose-API

**Comment by vishaldhiman** [comment]
Yes I added the instructions on the api webpage on github. Did you open the
api webpage?

Vishal

**Comment by vishaldhiman** [comment]
Here is the relevant information from the API webpage:

Upload Image

http://localhost:3002/restaurants/india-garden-199c2f32-f286-44a2-a563-bc30c6876b6f/menu_items/16130

Type: PUT (PUT is just like POST but is used for update)

Params:
- remotipart_submitted = true
- restaurant_id = 'india-garden-199c2f32-f286-44a2-a563-bc30c6876b6f'
  (This is also part of the URL)
- id = 16130 (This is the menu item id)
- menu_item[assets_attributes[0[file]]] = <>

Notes:
- Note that though on the website we are only allowing 1 image per dish,
  the index 0 in the file means that the system can allow multiple files
  using index 0,1,... - the client needs to take care which index to update.
  However, the search API only returns 1 image in the thumbnail key. *\* Only
  1 thumbnail per dish really speeds up the results.
- We can build a menu_item API call that can return all the ratings &
  images for a dish.
